app/controllers/pages_controller.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 class PagesController < ApplicationController
   def index
     render(layout: "layouts/homepage")
   end
 
-  def get_started; end
+  def start; end
```

Comment:
It would be good to understand what cop made this change.

Also I'm struggling to understand how this change doesn't break the start page.

Reply:
This is core Ruby style https://github.com/rubocop/rubocop/blob/master/lib/rubocop/cop/naming/accessor_method_name.rb

Reply:
Does this mean we can't make any pages that start with the word 'get'?